GHIElectronics.TinyCLR.Devices.UsbClient/Mouse.cs

```diff
@@ -287,43 +309,57 @@ public Mouse(UsbClientController usbClientController, UsbClientSetting usbClient
         /// <param name="stepY">The amount by which to increment the y position each request.</param>
         /// <param name="stepDelay">How long to wait between each step.</param>
         public void MoveCursorTo(int x, int y, int stepX, int stepY, int stepDelay) {
```

Comment:
Why not just use mouse.Move(x,y)

Reply:
MoveTo or MoveCursorTo is much more clear but I just changed as you asked.